### PR TITLE
Normalize date formatting and domain checking

### LINKED ISSUE #82: Normalize shared constants
Some values we use are the same among many services (e.g. timestamp formats) yet are reimplemented every time

we should have some sort of god of war package that can hold this

### PR DESCRIPTION
**Pull request by pajlada**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

- [x] internal/resolvers/default/resolver.go MaxTitleLength
- [x] internal/resolvers/default/resolver.go MaxDescriptionLength
- [ ] internal/resolvers/default/thumbnail.go maxThumbnailSize???

- [x] internal/resolvers/discord/load.go date formatting

- [x] internal/resolvers/imgur/check.go could use a better domain checker
- [x] internal/resolvers/imgur/helpers.go formatTimestamp makes a timestamp "created at" string from a unix time (int64)
- [x] internal/resolvers/imgur/resolver.go timestmapFormat

- [x] livestreamfails/load.go CreationData "created at" format
- [x] livestreamfails/resolver.go better domain checking

- [x] supinic/check.go better exact domain-checking
- [x] supinic/load.go better duration formatting?

- [x] twitch/load duration formattiong
- [x] twithc/load creationdate formatting
- [x] twitch/resolver domain checking

- [x] internal/resolvers/twitter/check.go domain checking
- [x] internal/resolvers/twitter/resolver.go timestampFormat should be "humanized"
      CHANGELOG!!!!!!!: twitter timestamp formatting has been changed from "Jan 2 2006" to "02 Jan 2006"

- [x] wiki/resolver max title length/descriptionlength/EXTRACT LENGTH??

- [x] internal/resolvers/youtube/resolver.go domain fixing
- [x] internal/resolvers/youtube/load.go durationParsing from string
- [x] internal/resolvers/youtube/load.go PublishDate/CreationDate

Fixes #82

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- Twitter resolver timestamp date format changed from `Jan 2 2006` to `02 Jan 2006`
```
Comment (zneix) [inline CHANGELOG.md:3]: Include PR number in the changelog entry
```suggestion
- Twitter resolver timestamp date format changed from `Jan 2 2006` to `02 Jan 2006` (#105)
```

**Review by zneix — CHANGES_REQUESTED** [review]

**Comment by pajlada** [comment]
@zneix please take a look at this LurkMode

I will be running this locally for a while to ensure it works as expected since we don't have many tests that would cover this

**Review by zneix — COMMENTED** [review]
Code seems good to me, however I will try this PR out on my instance of the API and see how it works.

**Comment by pajlada** [comment]
image

**Pull request merged**